packages/core/src/orchestrator.ts

```diff
@@ -56,7 +57,9 @@ export type EnvelopOrchestrator<
   getCurrentSchema: () => Maybe<GraphQLSchema>;
 };
 
-export function createEnvelopOrchestrator<PluginsContext = any>(plugins: Plugin[]): EnvelopOrchestrator<any, PluginsContext> {
+export function createEnvelopOrchestrator<PluginsContext extends DefaultContext>(
```

Comment:
I spent some time to understand the hook typings, I think my changes are correct now.
I changed the signature of `createEnvelopOrchestrator<PluginsContext = any>` to `createEnvelopOrchestrator<PluginsContext extends DefaultContext>` which makes the typings more clear.
If I understand this right, the `PluginsContext` refrects the type of the `contextValue` used in the current envelop instance, so it made sense to make it extend `DefaultContext`.